### ISSUE TITLE
Reproducing Results on UCR Archive's Fungi Dataset 

### ISSUE THREAD

**Issue by ghost** [Alrash/TimesURL#3]
Thanks for sharing your work. 

I conducted experiments on the UCR archive's Fungi dataset using your code, but the results were not reproducible. Regardless of whether the learning rate was set to 0.0001 (default value in your code) or 0.001, there was a difference compared to the reported accuracy.

Specifically, I have executed following commands: 
python -u train.py --dataset Fungi --loader UCR --batch-size 8 --repr-dims 320 --max-threads 8 --seed 42 --eval

Results with learning rate set to 0.0001: Evaluation result: {'acc': 0.7580645161290323, 'auprc': 0.8211251408603647} (lr=0.0001)
Results with learning rate set to 0.001: Evaluation result: {'acc': 0.9139784946236559, 'auprc': 0.9722505137346216} (lr=0.001)

Are there any additional steps or considerations I should be aware of?

Thank you for your help!
